components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/internal/OAuth2ServiceComponent.java

```diff
@@ -633,6 +634,29 @@ protected void unsetOrganizationUserResidentResolverService(
         OAuth2ServiceComponentHolder.setOrganizationUserResidentResolverService(null);
     }
 
+    @Reference(
+            name = "organization.mgt.service",
+            service = OrganizationManager.class,
+            cardinality = ReferenceCardinality.MANDATORY,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unsetOrganizationManagementService"
+    )
+    protected void setOrganizationManagementService(OrganizationManager organizationManagementService) {
+
+        if (log.isDebugEnabled()) {
+            log.debug("Setting the organization management service.");
```

Comment:
We can skip the debug log enable check as logging a string.

Reply:
fixed in [b2f63a2](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1910/commits/b2f63a2be4692c57df1211c7b83f1681ff71801a)